### DISCUSSION TITLE
关于SSR的问题 在F12网络面板中隐藏XHR/Fetch请求

### DISCUSSION THREAD

**Discussion by LuckyQ7** [nuxt/nuxt#29835] [Questions] [answered]
我使用  `useFetch` 去发出网络请求，在首次加载的时候 网络面板不会看到请求，但是在路由跳转的时候可以看到，我该怎么解决它

我想所用请求仅在服务端发出

**Comment by aozi0303** [comment]
the same question

**Comment by xjccc** [comment, marked as answer]
路由跳转就是vue的spa模式了。所以转换为客户端请求了。你如果不想用这种模式，点击跳转全连接的方式
